terraform/c100-application-deploy.tf

```diff
@@ -0,0 +1,16 @@
+module "c100-application-deploy" {
+  source     = "./modules/repository-collaborators"
+  repository = "c100-application-deploy"
+  collaborators = [
+    {
+      github_user  = "timja"
+      permission   = "write"
```

Comment:
```suggestion
      permission   = "Push"
```

Reply:
Bare with on this need to check it is correct

Reply:
would need to be lower case push

Reply:
script is saying to use uppercase atm, so that should be fixed :)
https://github.com/ministryofjustice/github-collaborators/runs/7559608135?check_suite_focus=true

```suggestion
      permission   = "push"
```

Reply:
good point will update that cheers

Reply:
you able to commit the fix here @NickWalt01  ?